sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -18,18 +20,30 @@
 
     private final DiscountDAO discountDAO;
     private final MapperConverter mapper;
+    private final ReviewDAO reviewDAO;
 
     @Override
     public DiscountDto getById(Long id) {
-        return discountDAO.findById(id)
+        DiscountDto discountR = discountDAO.findById(id)
```

Comment:
Bad naming, what is discountR? May be just discount or smth readable

Reply:
And one more thing. DAO layer can return entity object(s), we should use mapper only on service layer, no need to convert entity object between DAO and service layers

Reply:
DiscountServiceImpl s a service layer.